addon/components/flexberry-file.js

```diff
@@ -9,64 +9,34 @@ import { translationMacro as t } from 'ember-i18n';
 /**
   Flexberry file component.
 
+  Usage sample:
```

Comment:
Then example will not appear here:
![image](https://cloud.githubusercontent.com/assets/7602229/16448108/2c2621da-3e09-11e6-993d-367e7f2c11ba.png)